src/Lykke.Service.CandlesHistory/Startup.cs

```diff
@@ -80,7 +84,7 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                 builder.RegisterModule(new ApiModule(
                     marketType,
                     candlesHistory.CurrentValue,
-                    settings.CurrentValue.Assets,
+                    assetsSettings,
```

Comment:
why not candlesHistory.Assets you got it anyway. assetsSettings could be removed then